examples/README.md

```diff
@@ -1,8 +1,17 @@
 This folder contains there different examples as shown below
```

Comment:
Please add details for the examples that support distributed training

Reply:
I have added those details for IrisClassification example. We are using DDP for that example